testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/management/cli/UndeployTestCase.java

```diff
@@ -114,10 +134,22 @@ public void testLegacyUndeployWithAllServerGroups() throws Exception {
         checkExist(result, cliTestApp1War.getName(), ENABLED, ctx);
 
         // From serverGroup1 only, still referenced from sg2. Must keep-content
-        ctx.handle("undeploy --server-groups=" + sgOne + ' ' + cliTestApp1War.getName());
```

Comment:
Does this change backward compatibility for previous server version? I'm not sure because these tests doesn't exist earlier.

Reply:
in previous server version when the user did not specify all relevant server group on call to "undeploy", the deployment was just "disabled" in the mentioned server groups on the background as the content needs to be kept which I found unexpected; in this PR CLI appeals on user to specify all groups to undeploy